### PR TITLE
PR template

### PR DESCRIPTION
**Pull request by RondeauG**
Template for PRs and `HISTORY.rst`

### CODE REVIEW

File: .github/PULL_REQUEST_TEMPLATE.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
+<!-- Please ensure the PR fulfills the following requirements! -->
+<!-- If this is your first PR, make sure to add your details to the AUTHORS.rst! -->
+### Pull Request Checklist:
+- [ ] This PR addresses an already opened issue (for bug fixes / features)
+    - This PR fixes #xyz
```
Comment (Zeitsperre) [inline .github/PULL_REQUEST_TEMPLATE.md:5]:
```suggestion
  - This PR fixes #xyz
```

File: HISTORY.rst [hunk 2]
```diff
@@ -2,6 +2,32 @@
 History
 =======
 
+v0.2.0 (unreleased)
+-------------------
+Contributors to this version: Trevor James Smith (:user:`Zeitsperre`), Gabriel Rondeau-Genesse (:user:`RondeauG`).
```
Comment (Zeitsperre) [inline HISTORY.rst:7]: These sphinx-like directives (something I adapted/modified from Pangeo) require some code additions to the documentation configuration. If it's fine with you, I'll add them in this PR either today or tomorrow.

File: .github/PULL_REQUEST_TEMPLATE.md
Comment (Zeitsperre) [inline .github/PULL_REQUEST_TEMPLATE.md:1]: We've used this template in a bunch of places, so it would be great to get this into the cookiecutter!

**Review by Zeitsperre — APPROVED** [review]
Looks good! I wouldn't mind making some changes if there's no rush.

File: HISTORY.rst (hunk 2 above)
Comment (RondeauG) [reply HISTORY.rst:7]: No problem!

**Comment by Zeitsperre** [comment]
Looking good! Let's merge it.
image

**Pull request merged**